app/templates/events/view/tickets/attendees.hbs

```diff
@@ -0,0 +1,63 @@
+<div class="ui compact menu pull right floated">
+  <div class="ui simple dropdown item">
+    {{t 'Showing all records'}}
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item">Choice 1</div>
```

Comment:
Please add
- Filter by date
- Filter by ticktet
- Remove all filters 
instead of choice 1, 2, 3.